docs/guides/computed-properties.mdx

```diff
@@ -6,23 +6,39 @@ description: 'Use object getters and setters'
 
 # Computed Properties
 
-Valtio itself doesn't provide a functionality for computed properties,
-but you can emulate it with object getters and setters.
+In Valtio you can use object & class getters and setters to create computed properties.
 
-⚠️ Note: This is a very advanced usage.
+⚠️ Note: Getters in JavaScript are a more advanced feature of the language, so Valtio recommends using them with caution. That said, if you are a more advanced JavaScript programmer, they should work as you expect; see the "Note about this `this`" section below.
```

Comment:
@dai-shi Just updated this section + addressed your other comments, should be ready for re-review. Thanks!

Reply:
Thanks so much for taking my preference into account. Really appreciated.